common/log/tag/tags.go

```diff
@@ -537,14 +532,9 @@ func MaxLevel(lv int64) ZapTag {
 	return NewInt64("max-level", lv)
 }
 
-// ShardTransferAcks returns tag for ShardTransferAcks
-func ShardTransferAcks(shardTransferAcks interface{}) ZapTag {
-	return NewAnyTag("shard-transfer-acks", shardTransferAcks)
-}
-
-// ShardTimerAcks returns tag for ShardTimerAcks
-func ShardTimerAcks(shardTimerAcks interface{}) ZapTag {
-	return NewAnyTag("shard-timer-acks", shardTimerAcks)
+// ShardQueueAcks returns tag for shard queue ack levels
+func ShardQueueAcks(shardQueueAcks interface{}) ZapTag {
+	return NewAnyTag("shard-queue-acks", shardQueueAcks)
```

Comment:
could you double check this could actually print the content of the map, not just the pointer to the map.

Reply:
Yes, I tried locally. It will print the actual content of `shardInfo.QueueAckLevels`.
But the one downside of the currently implementation is that the map key is the task category ID instead of the name. 

I can change this function to take in category name as well and make it part of the tag key. Then each task category can have its own tag.